JenkinsfileImageTest

```diff
@@ -0,0 +1,111 @@
+properties(
+        [
+                buildDiscarder(logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '1000', daysToKeepStr: '', numToKeepStr: '1000')),
```

Comment:
I think 1000 is too high since we are saving qcow2 images. We need to make sure the last success is always there, but I think we can save very minimal failures